app/views/govuk_component/feedback.raw.html.erb

```diff
@@ -0,0 +1,110 @@
+          Yes <span class="visually-hidden">this page is useful</span>
+      <% end %>
+
+      <%= link_to contact_govuk_path, {
+        class: 'pub-c-feedback__prompt-link js-toggle-form js-page-is-not-useful',
+        data: {
+          'track-category' => 'manualFeedbackForm',
+          'track-action' => 'ffNoClick'
+          },
+          'aria-controls': 'page-is-not-useful',
+          'aria-expanded': false
+        } do %>
+          No <span class="visually-hidden">this page is not useful</span>
+      <% end %>
+
+      <%= link_to contact_govuk_path, {
+        class: 'pub-c-feedback__prompt-link pub-c-feedback__prompt-link--wrong js-toggle-form js-something-is-wrong',
+        data: {
+          'track-category' => 'manualFeedbackForm',
+          'track-action' => 'ffWrongClick'
+          },
+          'aria-controls': 'something-is-wrong',
+          'aria-expanded': false
+        } do %>
+          Is there anything wrong with this page?
+      <% end %>
+    </div>
+
+    <div class="pub-c-feedback__prompt-success js-prompt-success js-hidden" tabindex="-1">
+      Thanks for your feedback
+    </div>
+  </div>
+
+  <form action="/contact/govuk/problem_reports" id="something-is-wrong" class="pub-c-feedback__form js-feedback-form js-hidden" data-track-category="manualFeedbackForm" data-track-action="ffFormSubmit">
+    <a href="#" class="pub-c-feedback__close pub-c-feedback__js-show js-close-form" aria-controls="something-is-wrong" aria-expanded="true" role="button">Close</a>
+
+    <div class="pub-c-feedback__grid-row">
+      <div class="pub-c-feedback__column-two-thirds">
+        <div class="pub-c-feedback__error-summary pub-c-feedback__js-show js-hidden js-errors" tabindex="-1"></div>
+
+        <input type="hidden" name="url" value="<%= request.original_url -%>">
+        <input type="hidden" name="user_agent" value="<%= request.user_agent -%>">
+
+        <h2 class="pub-c-feedback__form-heading">Help us improve GOV.UK</h2>
+        <p id="feedback_explanation" class="pub-c-feedback__form-paragraph">Don't include personal or financial information like your National Insurance number or credit card details.</p>
+
+        <div class="pub-c-feedback__form-group js-form-group">
+          <label class="pub-c-feedback__form-label" for="what_doing">
+            What were you doing on this page?
+          </label>
+          <textarea id="what_doing" class="pub-c-feedback__form-field" name="what_doing" rows="4" aria-required="true" aria-describedby="feedback_explanation"></textarea>
+        </div>
+
+        <div class="pub-c-feedback__form-group js-form-group">
+          <label class="pub-c-feedback__form-label" for="what_wrong">
+            What is wrong with this page?
+          </label>
+          <textarea id="what_wrong" class="pub-c-feedback__form-field" name="what_wrong" rows="4" aria-required="true"></textarea>
+        </div>
+
+        <input class="pub-c-feedback__submit" type="submit" value="Submit">
```

Comment:
Any reason you aren’t using the button component here?

Reply:
None whatsover. Will add.